fluid-music/test/test-client.js

```diff
@@ -67,6 +93,24 @@ describe('fluid.Client (TCP/IPC Client)', function() {
     return result.should.be.rejected();
   }));
 
+  it('When server returns an invalid response, the promise returned by client.send should be rejected', (function() {
+    const client = new fluid.Client(22354);
+    const result = client.send(fluid.audiotrack.select('hi'));
+    return result.should.be.rejected();
+  }));
+
+  it('When the client sends an incomplete message, the promise should be rejected', (function() {
```

Comment:
Should this be "when the server replies with an incomplete message"?

Reply:
Both sides should be tested. (when the client sends an incomplete message, and when the server replies with an incomplete message)